codegen/smithy-python-codegen/src/main/java/software/amazon/smithy/python/codegen/ClientGenerator.java

```diff
@@ -395,10 +395,6 @@ async def _handle_attempt(
             writer.addImport("smithy_core", "URI");
             writer.write("""
                         # Step 7f: Invoke endpoint_resolver.resolve_endpoint
-                        if config.endpoint_resolver is None:
-                            raise $1T(
-                                "No endpoint_resolver found on the operation config."
-                            )
```

Comment:
Why did this check get removed? Is this one of the suppressions you referred to in the overview?